src/Types.tsx

```diff
@@ -0,0 +1,18 @@
+export type CreateRoom = { size: number }; // data sent by frontend on room creation information
```

Comment:
Just out of curiosity, why did you make this a tsx file?

Reply:
I was just trying to keep everything consistent for linting but now I see that it lints all other file types as well. Since it's not using any JSX-specific syntax we can change it if you want! I honestly just didn't have much experience with typescript.